src/ngraph/ngraph_graph.cc

```diff
@@ -365,8 +365,8 @@ void CollapseSubgraph(Graph* graph, int subgraph_num) {
     }
     for (auto node : tmpGraph->nodes_) {
       if (node->operation_ == "Convolution" ||
+          node->operation_ == "Deconvolution" ||
           node->operation_ == "BatchNorm" ||
-          node->operation_ == "SoftmaxOutput" ||
```

Comment:
Why'd you remove `"SoftmaxOutput"`?

Reply:
Not actually needed, Adam put it in and then worked around it.